index.js

```diff
@@ -190,6 +190,9 @@ exports.compileEVM = function (evmCode, stackTrace) {
         }
         i += op.number
         break
+      case 'PC':
+        wasmCode = `(i64.const ${i}) ${wasmCode}`
```

Comment:
Why is the PC i64? We limit codesize to i32 so we could limit this to i32 as well. (Probably better to do in a separate PR)